test/markdown_renderers.js

```diff
@@ -28,4 +30,22 @@ describe('apifyMarked custom renderer works', () => {
         expect(renderedTitle).to.equal(`
             <h3 id="welcome-title-id"><a href="#welcome-title-id"></a>Welcome to Apify</h3>`);
     });
+
+    it('replaces relative URLs from GitHub repos with absolute URLs in links', () => {
```

Comment:
Maybe add one test when the URL is an absolute link already, it has to stay the same.